SU2_CFD/src/numerics/scalar/scalar_sources.cpp

```diff
@@ -0,0 +1,930 @@
+/*!
+ * \file turb_sources.cpp
+ * \brief Implementation of numerics classes for integration of
+ *        turbulence source-terms.
```

Comment:
Again :+1: for taking the plunge on this.
I would not have moved the sources, I think these will always be very specific to each derived solver.
It would be better for navigation to keep it in the turbulence folder.
TBH I would not have moved anything that was not immediately necessary (all numerics in this case) because if someone comes along with changes to those classes....

Reply:
My motivation to move everything was more of a test/first-step to see if I break anything with that -> inserting the CScalar layer and moving things there. Turbulent sources will certainly move back and everything else specific will move back at some point.

> code this common interface and push all turb-specific things back into the respective files (that is why I didn't git mv )

I still have to figure out how a suitable generalized interface looks like and based on that I will reject the temporary residence permits for some stuff (e.g. all turb sources) 👍

Reply:
But by all means, I highly appreciate all sorts of comments as I there is lots of things to do wrong so I am glad for additional brains to look at this 🧠 👀

Reply:
Sure but you only need to worry about the solver and variables. The numerics are fairly independent (for now).
Minimize the number of changes per PR is all I'm saying. I think that's a guideline we all endorse :)

Reply:
> TBH I would not have moved anything that was not immediately necessary (all numerics in this case) because if someone comes along with changes to those classes....

well, yes that happenend ... should have listened directly